src/edit/handles/ScaleHandle.js

```diff
@@ -11,24 +11,24 @@ L.ScaleHandle = L.EditHandle.extend({
   },
 
   _onHandleDrag() {
-    var overlay = this._handled;
-    var map = overlay._map;
-    var edgeMinWidth = overlay.edgeMinWidth;
-    var formerLatLng = overlay.getCorner(this._corner);
-    var newLatLng = this.getLatLng();
-    var scale = this._calculateScalingFactor(formerLatLng, newLatLng);
+    let overlay = this._handled;
+    let map = overlay._map;
+    let edgeMinWidth = overlay.edgeMinWidth;
+    let formerLatLng = overlay.getCorner(this._corner);
+    let newLatLng = this.getLatLng();
+    let scale = this._calculateScalingFactor(formerLatLng, newLatLng);
 
     /*
      * checks whether the "edgeMinWidth" property is set and tracks the minimum edge length;
      * this enables preventing scaling to zero, but we might also add an overall scale limit
      */
 
     if (!edgeMinWidth) { edgeMinWidth = 50; } /* just in case */
-    var corner1 = map.latLngToLayerPoint(overlay.getCorner(0));
-    var corner2 = map.latLngToLayerPoint(overlay.getCorner(1));
-    var w = Math.abs(corner1.x - corner2.x);
-    var h = Math.abs(corner1.y - corner2.y);
-    var distance = Math.sqrt(w * w + h * h);
+    const corner1 = map.latLngToLayerPoint(overlay.getCorner(0));
+    const corner2 = map.latLngToLayerPoint(overlay.getCorner(1));
+    let w = Math.abs(corner1.x - corner2.x);
+    let h = Math.abs(corner1.y - corner2.y);
+    cpnst distance = Math.sqrt(w * w + h * h);
```

Comment:
```suggestion
    const distance = Math.sqrt(w * w + h * h);
```

Reply:
Thank you @TildaDares  for editing this I didn’t notice that I had made a typographical error

Reply:
Hi @leumasxxx, I haven't changed it yet. You can click the `Commit Suggestion` button to do that. Thanks!